src/sidebar/store/modules/filters.js

```diff
@@ -169,10 +169,21 @@ function changeFocusModeUser(user) {
  * @param {FilterOption} filterOption
  */
 function setFilter(filterName, filterOption) {
-  return {
-    type: actions.SET_FILTER,
-    filterName,
-    filterOption,
+  return (dispatch, getState) => {
```

Comment:
This is the other spot where the comments seem lame, and the underlying situation even lamer. Can discuss in person if it helps.